requre/helpers/tempfile.py

```diff
@@ -24,30 +24,53 @@
 import os
 import logging
 from typing import Optional
-from requre.storage import StorageCounter
+from requre.cassette import Cassette
+from requre.objects import ObjectStorage
 
 logger = logging.getLogger(__name__)
 
 
-class TempFile(StorageCounter):
+class TempFile(ObjectStorage):
     """
     replace system tempfile module with own predictable names implementation
      of temp files for mocking
     """
 
     root = "/tmp"
     prefix = "static_tmp"
+    _cassette: Cassette = None
+    _cassette_file = None
+    counter = 0
+    dir_suffix = "file_storage"
+
+    @classmethod
+    def set_cassette(cls, value: Cassette):
+        print(">>", value.storage_file, cls._cassette_file)
```

Comment:
Can we use logger instead so one can filter/manage from outside of this codebase?

Reply:
This was just for my debugging here for me, to see changes. Removed :-)